contracts/src/SP1ICS07Tendermint.sol

```diff
@@ -135,9 +142,20 @@ contract SP1ICS07Tendermint is
 
     /// @notice The entrypoint for misbehaviour.
     /// @inheritdoc ILightClient
-    function misbehaviour(bytes calldata) public pure {
-        // TODO: Not yet implemented. (#56)
-        revert FeatureNotSupported();
+    function misbehaviour(bytes calldata misbehaviourMsg) public {
```

Comment:
We could do this, but it would require some changes in this repo and the solidity one because it requires changing the ILightClient interface. Perhaps we could take a look at the ILightClient interface and see if there are other changes too and do them all over the place at once (in a separate issue?)

Reply:
Yes, I agree it's better to do this at a later stage and bundle with potentially other changes.